src/test/scala/spray/json/JsonParserSpec.scala

```diff
@@ -73,11 +73,16 @@ class JsonParserSpec extends Specification {
       JsonParser(json.prettyPrint.getBytes("UTF-8")) === json
     }
     "be reentrant" in {
+      import scala.concurrent.{Await, Future}
+      import scala.concurrent.duration._
+      import scala.concurrent.ExecutionContext.Implicits.global
+
       val largeJsonSource = scala.io.Source.fromInputStream(getClass.getResourceAsStream("/test.json")).mkString
-      import scala.collection.parallel.immutable.ParSeq
-      ParSeq.fill(20)(largeJsonSource).map(JsonParser(_)).toList.map {
-          _.asInstanceOf[JsObject].fields("questions").asInstanceOf[JsArray].elements.size
-      } === List.fill(20)(100)
+      val list = Await.result(
+        Future.traverse(List.fill(20)(largeJsonSource))(src => Future(JsonParser(src))),
```

Comment:
use `Future` instead of parallel-collections because parallel-collections for Scala 2.13.0-M4 does not available yet 😕 

https://github.com/scala/scala-parallel-collections/issues/41

Reply:
Good, we're not in love with the par collections anyway :)